com.unity.multiplayer.mlapi/Runtime/Logging/FileLogger.cs

```diff
@@ -0,0 +1,38 @@
+using System.IO;
+
+namespace MLAPI.Logging
+{
+    /// <summary>
+    /// Helper class for logging
+    /// Saves to a file identified by the process ID of each client
+    /// This is useful when debugging what happens on multiple machines
+    /// ! Only recommended for debugging, not for production !
+    /// </summary>
+    public class FileLogger
+    {
+        public static FileLogger Get() { return instance; }
+        private static FileLogger instance = new FileLogger();
+        private StreamWriter writer;
+
+        private FileLogger()
+        {
+            System.Diagnostics.Process p = System.Diagnostics.Process.GetCurrentProcess();
```

Comment:
Might you want to bail if on iOS / Android / Console?  I wonder what happens if you try to write to files via Unity

Reply:
Here is one example that "should" work as the "Assets/Resources/<filename>.txt" should be valid.
https://www.codegrepper.com/code-examples/csharp/save+text+file+unity

Reply:
yeah, I'm also curious how portable this API really is. if not very portable, we might want to wrap it with platform macros.